index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <h1 id="title">Dr. Peter</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.pixabay.com/photo/2015/01/08/18/29/entrepreneur-593358_960_720.jpg"
+          alt="Dr.Peter sitting"
+        />
+        <figcaption id="img-caption">we are good</figcaption>
+      </div>
+      <section id="tribute-info">
+        <h3>Here's a timeline of Dr. Nabeel's life</h3>
+        <ul>
+          <li>
+            <strong>1983</strong>
+            Born in Lahore Pakistan
+          </li>
```

Comment:
- [ ] You also made use of the right semantics to display the year and place of birth, You've done very well.

Reply:
- [ ] It is good to mention what the author of the code did well 👍️ but I would suggest you point out things like : 
  - Lack of [semantic HTML tags use](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
  - Not applying [HTML & CSS best practices correclty](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
  - Bugs missed the first time the project was built.

- [ ] And as requested by the previous reviewer do not forget to explain `why` when you request a change because it helps the author of the code, not repeat the mistake again.
  - An example of a clear comment would be 👇️
  
  ``` The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional? ```

Reply:
@ridwanediallo, i really do not understand the point you are trying to make. 

The said html code was written on the direction of freecodecamp during microverse trial session. There's literally no issue with the code. 

I guess you are misunderstanding the previous reviewer. Well, he only wanted to see that i can make an inline comment which i did.

Reply:
Hello @PeteUgwu, I am sorry if you did not understand what I meant. You are right the previous reviewer asked to use the inline comment but again he said ` Don't forget to keep your comment polite, explain why you ask for the changes and have a clear comment too.`  You are supposed to request some changes. It's true the code was written under the direction of freecodecamp during microverse trial session. However, you can intentionally introduce some bugs or leave a commented code, or stead of a `p` tag use something else for example. And request a change in a polite way and explain why we should not for instance have a commented code in our codebase.

Reply:
I tried to reach out to you on slack and on zoom but I could not find you. Please if you are still confused, reach out to me on slack `@Ridwan` or call me on zoom.

Reply:
Fixed, Thanks for your time.